### DISCUSSION TITLE
Syndicated news and autocue

### DISCUSSION THREAD

**Discussion by mishcazzulani** [AzuraCast/AzuraCast#7449] [Q&A]
Hello, I'm happily using autocue on azuracast stable v0.20.2 with the default settings (I've just enabled it in the ui, no liquidsoap tweaking).

My only problem arises with syndicated news that I download half an hour before airtime with a wget batch which always overwrites the same file (so no liq_ tags in it).

The first time it aired autocue compiled azuracast's track editor fields and now every time it goes onair it's played with wrong cue points.

I see in the faq you suggest to add the liq_cue_file tag and set it to true but how to do it?

Is it better to cue the file in my script like this:

echo '{"liq_cue_file": true}' | cue_file -j - -fwr "news.mp3"

or to delete the metadata with an api call to force autocue:

curl -X 'PUT'
'https://xxx.xxx.xxx/api/station/x/file/xxxx'
-H 'accept: application/json'
-H 'X-API-Key: xxx'
-H 'Content-Type: application/json'
-d '{
"extra_metadata": {
"liq_amplify": null,
"liq_cross_start_next": null,
"liq_cue_in": null,
"liq_cue_out": null,
"liq_fade_in": null,
"liq_fade_out": null
}
}'

**Comment by LordHelmchen666** [comment]
It is up to you which style you prefer. It is always same approach.
Yes you could use to tag file `echo '{"liq_cue_file": true}' | cue_file -j - -fwr "news.mp3"`. 

If you want to change more tags mid3v2 is also good to go. For example
`mid3v2 -a "ARTIST" -t "TITLE" -A "ALBUM" -g "GENRE" --TXXX "liq_cue_file:true" news.mp3`

**Comment by LordHelmchen666** [comment]
With the new information from here https://github.com/AzuraCast/AzuraCast/discussions/7466#discussioncomment-10743661 my solution is 

1. Download the file
2. Get duration from file with ffprobe (set "liq_cue_out" and "liq_cross_start_next" to full duration. That way it will not "cut off")
3. Tag it with cue_file (Autocue)
4. Change some tags with mid3v2

```
    curl https://xxx:xxx@api.radioadmin.laut.fm/news/1 --output $directory/lautfm-news.mp3 
    LENGTH=$(ffprobe -show_entries format=duration -of default=noprint_wrappers=1:nokey=1 $directory/lautfm-news.mp3)
    echo '{"liq_fade_in": 0.1, "liq_fade_out": 0.1, "liq_cue_out": '$LENGTH', "liq_cross_start_next": '$LENGTH'}' | cue_file -j - -fkwrn -- $directory/lautfm-news.mp3
    mid3v2 -a "laut.fm" -t "Nachrichten" -A "laut.fm Nachrichten" -g "News" $directory/lautfm-news.mp3
```

EDIT. I am on latest Rolling Release and deleted all "extra metadata"

**Reply by Marcioghiraldelli** [reply]
Do you put that inside tls_config?

**Reply by mishcazzulani** [reply]
@LordHelmchen666 Thank you very much! Do you execute it inside docker or outside?

@Marcioghiraldelli What is tls_config?

**Reply by LordHelmchen666** [reply]
@mishcazzulani I ran this outside docker directly on host system.

@Marcioghiraldelli This is a snippet of a bash script. I ran a cron job every minute. In the script I check the time e. g. if it is 53 minute of the hour (to download the news track) and then execute this commands.
